reporters/influx/influx.go

```diff
@@ -0,0 +1,150 @@
+package influx
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	client "github.com/influxdata/influxdb/client/v2"
+	"gopkg.in/vinxi/metrics.v0"
+)
+
+// Config stores the InfluxDB connection params.
+type Config struct {
+	URL      string
+	Database string
+	Username string
+	Password string
+	Tags     map[string]string
+}
+
+// Reporter implements an InfluxDB metrics reporter who send data to a InfluxDB server via HTTP.
+type Reporter struct {
+	config Config
+	client client.Client
+}
+
+// New creates a new InfluxDB reporter which will post the metrics to the specified server.
+func New(c Config) *Reporter {
+	re := &reporter{config: c}
+	if err := re.makeClient(); err != nil {
+		log.Printf("unable to make InfluxDB client. err=%v", err)
```

Comment:
`New()` is designed to be consumed directly by `metrics.New(...Reporter)`. 
Having multiple return values makes that impossible, but handling errors.

> When it cannot make a client, subsequent call to Report will break, right?

Should fail with a `nil` reference error.

How to improve this?
Error handling should be a priority, and if something fails, it should make noise. 

I just modified the `Reporter` interface to return an error, making possible to deal with that programmaticaly in the future from the `metrics` package (perhaps binding some error-specific handler):
```go
type Reporter interface {
	Report(Report) error
}
```

What do you think? Other ideas?